### PR TITLE
Create blog-contribution-guidelines

### PR DESCRIPTION
**Pull request by jsalinas29**
/kind documentation
how to contribute to the blog

### CODE REVIEW

File: blog-contribution-guidelines [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* Articles about Falco plugins
+* Articles about how to write Falco rules
+* Industry insight about container security and how to use Falco
+* Project updates about Falco
+* Reposting is okay, but we will need to do some editing to avoid SEO chaos
+
+**Editorial calendar**
+The goal of the editorial calendar is to proactively plan, source speakers, and curate content to the Falco community in a transparent manner. Please review the editorial calendar to understand when your blog will be published.
+
+[**Editorial calendar**](https://)
```
Comment (Andreagit97) [inline blog-contribution-guidelines:29]: this link is broken (`https://`)

File: blog-contribution-guidelines [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+**Submit your PR**
+Now that you understand the content guidelines, editorial calendar, and publishing process you are ready to submit your pull request. All blog content must be submitted through this process to be considered for publication. 
+* How to submit your first PR blog in 5 easy steps
+* Step 1: Fork to the [falco-website repo](https://github.com/falcosecurity/falco-website) on Github
+* Step 2: Clone locally your branch and create a new branch
+* Step 3: Create your blog post in markdown format, [see other posts as example](https://github.com/falcosecurity/falco-website/tree/master/content/en/blog)
+* Step 4: Add images to the [img folder](https://github.com/falcosecurity/falco-website/tree/master/static/img)
+* Step 4: Commit your work with a signoff and push to Github
+* Step 5: Create your pull request to upstream repo with your changes
+Filing your first PR? Go here 👉 [for step by step instructions ](https://)
```
Comment (Andreagit97) [inline blog-contribution-guidelines:66]: this link is broken (`https://`)

File: blog-contribution-guidelines [hunk 3]
```diff
@@ -0,0 +1,78 @@
+## How to get started contributing to the Falco blog
+
+Welcome to the Falco community blog. This blog serves as an educational resource for community members, maintainers, and contributors. 
+
+The process is meant to be self-serve but if you experience any difficulties reach out to us at contact us via the [Kubernetes Slack on the #falco](https://kubernetes.slack.com/messages/falco) channel. 
+
+**TLDR;** submit your [blog PR](https://github.com/falcosecurity/falco-website/pulls) here.
```
Comment (Andreagit97) [inline blog-contribution-guidelines:7]:
```suggestion
**TLDR;** submit your blog PR [here](https://github.com/falcosecurity/falco-website/pulls).
```
I would set the link on the `here` word, feel free to change it if you don't like it :)

**Review by Andreagit97 — COMMENTED** [review]
As said in #178 i would add to this file the `.md` extension

File: blog-contribution-guidelines [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* Articles about Falco plugins
+* Articles about how to write Falco rules
+* Industry insight about container security and how to use Falco
+* Project updates about Falco
+* Reposting is okay, but we will need to do some editing to avoid SEO chaos
+
+**Editorial calendar**
+The goal of the editorial calendar is to proactively plan, source speakers, and curate content to the Falco community in a transparent manner. Please review the editorial calendar to understand when your blog will be published.
+
+[**Editorial calendar**](https://)
```
Comment (jsalinas29) [inline blog-contribution-guidelines:29]:
```suggestion
[**Editorial calendar**](https://docs.google.com/spreadsheets/d/1LiE5yE3kplD_Kg3y1YVZphCiChmqC2xynMJLoqlREPM/edit?usp=sharing)
```

File: blog-contribution-guidelines (hunk 2 above)
Comment (jsalinas29) [reply blog-contribution-guidelines:66]:
```suggestion
Filing your first PR? Go here 👉 [for step by step instructions ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
```

**Comment by jsalinas29** [comment]
@Andreagit97 not sure if i updated the file name correctly - still learning gh 😝

File: blog-contribution-guidelines (hunk 4 above)
Comment (Andreagit97) [reply blog-contribution-guidelines]: done :)

**Comment by Andreagit97** [comment]
> @Andreagit97 not sure if I updated the file name correctly - still learning gh 

don't worry I've just renamed it for you :) what I meant was to rename the file locally from `blog-contribution-guidelines` to `blog-contribution-guidelines.md`

**Review by leogr — APPROVED** [review]
